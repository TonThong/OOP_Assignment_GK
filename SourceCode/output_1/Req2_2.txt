CruiseShip [26, Castle CruiseShip, 120 Palm St, City B, true, true, true, true, false]
Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
Resort [11, Sandy Beach Resort, 4, 234 Sandy Bay, false, City B]
Hotel [22, 4, White Palace Hotel, 78 Queen St, City B]
Hotel [22, 4, White Palace Hotel, 78 Queen St, City B]
