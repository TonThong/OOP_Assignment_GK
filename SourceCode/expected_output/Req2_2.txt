Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
Hotel [2, 5, Luxury Hotel, 456 Elm St, City B]
